source/brailleDisplayDrivers/handyTech.py

```diff
@@ -524,6 +525,20 @@ class Modular80(Modular):
 	numCells = 80
 
 
+class Activator(TimeSyncFirmnessMixin, AtcMixin, JoystickMixin, TripleActionKeysMixin, Model):

```

Comment:
I thought the Activator supported HID. Is there something in the HandyTech protocol that isn't supported by your HID implementation?

Reply:
The Activator supports both Braille HID and the Handy Tech Braille protocol. Several features, including time syncing with the computer, are not part of the HID standard and are therefore only supported in the Handy Tech protocol. Additionally, Braille HID for Activator may not be available via Bluetooth, in which case the Handy Tech Braille protocol must be used.
Handy Tech has been renamed to Help Tech, but the protocol pre-dates this.